src/main/java/hudson/plugins/sidebar_link/ProjectLinks.java

```diff
@@ -55,13 +55,17 @@ public ProjectLinks(List<LinkAction> links) {
 
     public List<LinkAction> getLinks() { return links; }
 
-    @Override
     public Collection<? extends Action> getJobActions(AbstractProject<?,?> job) {
         if(links == null)
             return new ArrayList<LinkAction>();
         return links;
     }
 
+    @Override
+    public Collection<? extends Action> getJobActions(Job<?,?> job) {
+        return Collections.EMPTY_SET;
```

Comment:
I also do not understand why you would need to return Empty list at all. Once may add actions for Pipeline jobs as well.